core/lib/web3_decl/src/client/network.rs

```diff
@@ -55,7 +61,7 @@ impl From<L2ChainId> for L2 {
 
 /// Associates a type with a particular type of RPC networks, such as Ethereum or ZKsync Era. RPC traits created using `jsonrpsee::rpc`
 /// can use `ForNetwork` as a client boundary to restrict which implementations can call their methods.
-pub trait ForNetwork {
+pub trait ForEthereumLikeNetwork {
```

Comment:
Dumb question: What's the reason for this renaming? AFAIU, we still don't have non-"Ethereum-like" networks, so why complicate the trait name?

Reply:
Imo it's easier to understand what this does now without having to see definition

Reply:
@slowli Just visually it seemed to me that `ForNetwork` should implement only things that could be used for any network. But right now there appeared to be types that are implemented for all networks there, like 

```
/// Subset of the L1 `eth` namespace used by the L1 client.
#[rpc(client, namespace = "eth", client_bounds(Self: ForEthereumLikeNetwork))]
```

So that's why I decided that rename would be necessary to ensure that people understand this network is expected to be Ethereum-like